flower-shop/flower_shop.py

```diff
@@ -0,0 +1,67 @@
+flowers_list = []
+bouquets = []
+order_list = []
+
+
+class Flower:
+    def __init__(self, name, color):
+        self.name = name
+        self.color = color
```

Comment:
Use a enum sequence to store flower types rather than defining an attribute:
more about enums here: https://docs.python.org/3/library/enum.html

Reply:
Add an ability to compare Flower's (are they equal or not, hint: `__eq__`)

Could it be like this; if Flower's types are different then they're not equal. If two flowers have equal flower types then they're equal.

Reply:
Add `__str__` methods to classes